spec/peep_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require 'helpers/user_helpers'
+
+describe Peep do
+
+  context 'can create a' do
```

Comment:
This is not the recommended way to use contexts.  Context is intended to provide a block in which to place tests with similar conditions, so that they can share `let`s and `before`s